Components/Modal/modal.css

```diff
@@ -0,0 +1,56 @@
+.modal-container {
+  background-color: var(--lightGray-color);
+  min-height: 20rem;
+  border: 1rem solid var(--lightGray-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white-color);
+  width: 32rem;
+  min-height: 23rem;
+  border-radius: 0.5rem;
+}
+
+.modal-heading {
+  padding: 1rem;
+  padding-bottom: 0rem;
+  padding-top: 2rem;
```

Comment:
you can give padding in a single line

Reply:
Thanks  [nisargpawade](https://github.com/nisargpawade) for the review